indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -0,0 +1,150 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+import itertools
+import json
+
+from uuid import uuid4
+
+from flask import session, request, redirect, jsonify
+from sqlalchemy.orm import joinedload
+
+from indico.core.db import db
+from indico.modules.events.registration.controllers.management import RHManageRegFormBase, RHManageRegistrationBase
+from indico.modules.events.registration.models.items import RegistrationFormItemType, RegistrationFormItem
+from indico.modules.events.registration.models.registrations import Registration, RegistrationData
+from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
+from indico.modules.events.registration.views import WPManageRegistration
+from indico.web.flask.templating import get_template_module
+from indico.web.flask.util import url_for
+from indico.web.util import jsonify_data
+from MaKaC.common.cache import GenericCache
+
+
+cache = GenericCache('reglist-config')
+
+
+def _get_filters_from_request():
+    filters = {}
+    for key in request.form.iterkeys():
```

Comment:
and to distinct the fields that have radioitems, is the
`field.input_type in ['radio', 'country']` enough?